_pages/community/events/hpca-2023.md

```diff
@@ -7,7 +7,9 @@ permalink: events/hpca-2023
 
 <iframe width="560" height="315" src="https://www.youtube.com/embed/npbMXXoKltg" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
 
-Slides: [PDF](/assets/files/hpca2023-tutorial/gem5-tutorial-hpca-2023.pdf), [Powerpoint](/assets/files/hpca2023-tutorial/gem5-tutorial-hpca-2023.pptx).
+Slides: 
+- [PDF](/assets/files/hpca2023-tutorial/gem5-tutorial-hpca-2023.pdf), [Powerpoint](/assets/files/hpca2023-tutorial/gem5-tutorial-hpca-2023.pptx).
```

Comment:
Maybe this is a dumb question @ivanaamit , but are the slides for the non-GPU sections not posted?  I don't see them anywhere on this md page -- maybe they are all in the codespace?

Separately, I think we can just post the PDF, no need for PPTX too.

For name, I think "GPU" is fine!

Reply:
I fixed it, the previous "PDF" link was for the non-GPU slides.